### PR TITLE
Show call length during a call

### PR DESCRIPTION
**Pull request by SimonBrandner**
Fixes https://github.com/vector-im/element-web/issues/18566
**Requires https://github.com/matrix-org/matrix-js-sdk/pull/1873**
Type: enhancement

<hr>

Screenshot_20210827_162534

---
Here's what your changelog entry will look like:

## ✨ Features
 * Show call length during a call ([\#6700](https://github.com/matrix-org/matrix-react-sdk/pull/6700)). Fixes vector-im/element-web#18566. Contributed by [SimonBrandner](https://github.com/SimonBrandner).

Preview: https://6128fa0f81174fe8c5f82b09--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: src/DateUtils.ts [hunk 1]
```diff
@@ -136,6 +136,18 @@ export function formatCallTime(delta: Date): string {
     return output;
 }
 
+export function formatSeconds(inSeconds: number): string {
```
Comment (turt2live) [inline src/DateUtils.ts:139]: We have a `<Clock />` component which does this for you (might need light edits to support hours though)

**Review by turt2live — COMMENTED** [review]
otherwise looks good to me - thanks! Has design reviewed this out of band somewhere?

File: src/DateUtils.ts (hunk 1 above)
Comment (SimonBrandner) [reply src/DateUtils.ts:139]: I am using that component already :P See [`49defd7` (#6700)](https://github.com/matrix-org/matrix-react-sdk/pull/6700/commits/49defd7b5a52087227ecb21834ddd098dababc2c) and [`c6400c8` (#6700)](https://github.com/matrix-org/matrix-react-sdk/pull/6700/commits/c6400c8db34e2384ce14ec461c04c99d3c8f3852)

Comment (turt2live) [reply src/DateUtils.ts:139]: huh, I'm clearly losing my mind. I thought `DateUtils` was a component 🤦‍♂️

**Review by turt2live — APPROVED** [review]

**Review by gaelledel — APPROVED** [review]

**Pull request merged**